paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -837,21 +783,6 @@ internal class DefaultFlowController @Inject internal constructor(
         return viewModel.state?.config?.appearance ?: PaymentSheet.Appearance()
     }
 
-    class BacsMandateException(
-        val type: Type
-    ) : Exception() {
-        override val message: String = when (type) {
-            Type.MissingInformation ->
-                "Bacs requires the account's name, email, sort code, and account number be provided!"
-            Type.IncorrectSelection -> "Cannot confirm non-Bacs payment method with Bacs mandate"
-        }
-
-        enum class Type {
-            MissingInformation,
-            IncorrectSelection
-        }
-    }
-
```

Comment:
Think I added this to be more specific about Bacs mandate errors but wasn't used in `PaymentSheet`, maybe we introduce it as an error type in `IntentConfirmationHandler`?